source/css/sass/_normalize.scss

```diff
@@ -51,47 +51,64 @@ h6 {
   @include h6-spacing();
 }
 
-p {
-  @include body-para();
+address,
+article,
+aside,
+blockquote,
+caption,
+details,
+dl,
+fieldset,
+figcaption,
+figure,
+footer,
+form,
+header,
+hr,
+main,
+nav,
+ol,
+p,
+pre,
+section,
+table,
+ul {
+  @include block-spacing();
+}
+
+figure {
+  @include margin(0, "inline");
+}
+
+dl,
+ol,
+ul {
+  dd > &,
+  li > & {
+    @include block-spacing(0);
```

Comment:
Just checking removing `@include body-typography()` intended.

Reply:
Would need to check in a browser, but I'm assuming they don't set typography on lists? 🤔

Reply:
Looking into this, seems underlying code originally came from eLife codebase. Happy to leave it out now for leaner code and address any issues arising as and when. Should be easy to spot as either list typography will break or it won't.